src/common/registry/storage/operator.entity.ts

```diff
@@ -39,4 +41,8 @@ export class RegistryOperator {
 
   @Property()
   usedSigningKeys!: number;
+
+  @PrimaryKey()
+  @Property({ length: 42 })
+  moduleAddress!: string;
```

Comment:
probably, we can create a foreign key for `srmodule_storage` table instead of having only module address? It will useful for joining tables.

Reply:
decided to move out to a separate indexing service